src/components/lab/ListItemView/ListItemView.tsx

```diff
@@ -0,0 +1,199 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+
+import * as React from 'react';
+
+import {Check} from '@gravity-ui/icons';
+import {focusable} from 'tabbable';
+
+import {useForkRef} from '../../../hooks';
+import {ArrowToggle} from '../../ArrowToggle';
+import {Button} from '../../Button';
+import {Icon} from '../../Icon';
+import type {DOMProps} from '../../types';
+import {block} from '../../utils/cn';
+import {filterDOMProps} from '../../utils/filterDOMProps';
+
+import './ListItemView.scss';
+
+const b = block('lab-list-item-view');
+
+export interface ListItemViewProps<T extends React.ElementType = 'div'> extends DOMProps {
+    id?: string;
+    children: React.ReactNode;
+    size?: 's' | 'm' | 'l' | 'xl';
+    selected?: boolean;
+    active?: boolean;
+    onClick?: (e: React.MouseEvent) => void;
+    disabled?: boolean;
+    selectionStyle?: 'check' | 'highlight' | 'none';
+    collapsible?: boolean;
+    collapsed?: boolean;
+    onCollapseChange?: (collapsed: boolean) => void;
+    draggable?: boolean;
+    nestedLevel?: number;
+    startContent?: React.ReactNode;
+    description?: React.ReactNode;
+    endContent?: React.ReactNode;
+    isContainer?: boolean;
+    component?: T;
+}
+
+export const ListItemView = React.forwardRef(ListItemViewComponent) as <
+    T extends React.ElementType = 'div',
+>(
+    props: ListItemViewProps<T> & Omit<React.ComponentPropsWithRef<T>, keyof ListItemViewProps<T>>,
+) => React.ReactElement;
+
+export function ListItemViewComponent(
+    props: ListItemViewProps & Omit<React.ComponentPropsWithoutRef<'div'>, keyof ListItemViewProps>,
+    ref: React.ForwardedRef<HTMLDivElement>,
+) {
+    const {
+        size,
+        active,
+        selected,
+        disabled,
+        onClick,
+        selectionStyle,
+        className,
+        style,
+        collapsed,
+        onCollapseChange,
+        children,
+        isContainer = false,
+        component: Component = 'div',
+        collapsible: _collapsible,
+        description,
+        draggable: _draggable,
+        startContent: _startContent,
+        endContent: _endContent,
+        nestedLevel: _nestedLevel,
```

Comment:
Let's use `Prop` suffix instead. `_` prefix has a special meaing in TS files

Reply:
This special meaning is used intentionally.